code/workspaces/web-app/src/components/stacks/StackCard.js

```diff
@@ -82,13 +83,13 @@ function styles(theme) {
     },
     shareStatus: {
       color: theme.typography.colorLight,
-      fontSize: theme.typography.body2.fontSize,
+    //  fontSize: theme.typography.body2.fontSize,
```

Comment:
commented code

Reply:
Just noticed that yesterday as you commented I think!